### ISSUE TITLE
indent: `use`

### ISSUE THREAD

**Issue by jbodah** [elixir-lang/vim-elixir#294]
# Actual

```
defmodule DeadboltTest do
  use ExUnit.Case
      doctest Deadbolt

      hello

end
```

# Expected

```
defmodule DeadboltTest do
  use ExUnit.Case
  doctest Deadbolt

  hello

end
```

**Comment by jbodah** [comment]
Waiting on fix to be pushed

**Comment by jbodah** [comment]
Fixed in 222381d
